spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -16,7 +16,7 @@ class TunesClient < Spaceship::Client
     # Legacy support
     ITunesConnectError = Tunes::Error
     ITunesConnectTemporaryError = Tunes::TemporaryError
-    ITunesConnectPotentialServerError = Tunes::PotentialServerError
+    ITunesConnectFlakyCallPotentialServerError = Tunes::PotentialServerError
```

Comment:
What is the purpose of this rename? Also, This may break people's code if they rescue `ITunesConnectPotentialServerError` and now it is undefined.

Reply:
The thinking behind the rename was that it describes the exception better - this can only be raised by "flaky calls" and doesn't really describe a server error, to be honest not even a potential one. 

The actual problem is probably a level deeper, in the upload of assets that are failing (and not being handled) - which then triggers the "flaky call" to return an error. If I figure out what _really_ is the other problem, and it is the sole reason for this here being triggered, I was thinking about just removing this.

Reply:
But I agree, the rename is problematic - and doesn't really help anyone here. I will remove it.

Reply:
Removed that commit and force pushed. Thanks for checking me on this! 👍